src/events/http.js

```diff
@@ -307,21 +369,25 @@ function httpWrapper(wrappedFunction) {
                 });
 
                 clientRequest.on('response', (res) => {
-                    res.on('data', (chunk) => {
-                        if (!chunk) {
-                            // Skip empty data
-                            return;
-                        }
-                        const totalSize = chunks.reduce((total, item) => item.length + total, 0);
-                        if (totalSize + chunk.length <= MAX_HTTP_VALUE_SIZE) {
-                            chunks.push(typeof chunk === 'string' ? Buffer.from(chunk) : chunk);
-                        }
-                    });
+                    // Listening to data only if options.epsagonSkipResponseData!=true or no options
+                    if (!options || (options && !options.epsagonSkipResponseData)) {
+                        res.on('data', (chunk) => {
```

Comment:
this looks like shared logic that can be extracted

Reply:
👍